aip/general/0193.md

```diff
@@ -98,7 +106,8 @@ does not exist, the service **must** error with `NOT_FOUND` (HTTP 404).
 <!-- prettier-ignore-start -->
 [aip-4221]: ../client-libraries/4221.md
 [details]: https://github.com/googleapis/googleapis/blob/master/google/rpc/error_details.proto
-[ErrorInfo]: https://github.com/googleapis/googleapis/blob/master/google/rpc/error_details.proto#L51
+[ErrorInfo]: https://github.com/googleapis/googleapis/blob/6f3fcc058ff29989f6d3a71557a44b5e81b897bd/google/rpc/error_details.proto#L27-L76
+[PreconditionFailure]: https://github.com/googleapis/googleapis/blob/6f3fcc058ff29989f6d3a71557a44b5e81b897bd/google/rpc/error_details.proto#L139-L166
```

Comment:
nit: should we start always referring to specific hashes? I feel like that's a good idea.

It's just a different pattern than the other references.

Reply:
I think so b/c ErrorInfo moved (in the file) at one point so the link was just wrong.

Reply:
FWIW I love the pattern. The only improvement would be if we could tie it to a semantically meaningful tag vs the sha.

But good call! I'll do the same in my changes.

Reply:
Great idea! I'll do that ASAP.

Reply:
Turns out github.com/googleapis/googleapis does not have any tags. I looked into what we be involved in having copybara create them. It's non-trivial afaict.